packages/tempus/src/Tempus_StepperRKBase.hpp

```diff
@@ -21,13 +24,16 @@ class StepperRKBase
 
 public:
 
-  virtual int getStageNumber() { return stageNumber_; }
+  virtual int getStageNumber() const { return stageNumber_; }
 
   virtual void setStageNumber(int s) { stageNumber_ = s; }
 
+  virtual Teuchos::RCP<Thyra::VectorBase<Scalar> > getStageX() {return stageX_;}
```

Comment:
I like moving this down to the base class. Can we move it further down so that we can unify getting this across the steppers? The default would be to either do nothing or throw an exception so that the user know that they are using this incorrectly.

Reply:
@krcb I would not move RK specific functions and data into the Stepper class.  I hear you about wanting to unify across steppers but I think it will cause more confusion for other non-RK steppers.  It seems the driver to unify is the new AppActions, but they will be specific to each stepper, so unifying would not gain much.